cmd/ingester/app/flags.go

```diff
@@ -85,6 +90,10 @@ func AddFlags(flagSet *flag.FlagSet) {
 		ConfigPrefix+SuffixEncoding,
 		DefaultEncoding,
 		fmt.Sprintf(`The encoding of spans ("%s" or "%s") consumed from kafka`, EncodingProto, EncodingJSON))
+	flagSet.String(
```

Comment:
why is it a string and not a number?

Reply:
That's a great question but I based this snippet on the parallelism flag above (also strangely a string). I can use `Float64` instead.